dandi/metadata.py

```diff
@@ -776,6 +776,7 @@ def process_ndtypes(asset, nd_types):
     technique = set()
     variables = set()
     for val in nd_types:
+        val = val.split()[0]
```

Comment:
You should first check that `val` is neither empty nor all-whitespace in order to avoid failures on indexing.

Reply:
i don't think nd_types can be empty or all whitespace by definition of pynwb. we can put checks, but i don't like necessarily checking things like this.